src/components/Features.jsx

```
@@ -0,0 +1,115 @@
+        <section id="features">
+            <h2>See it all in a new light.</h2>
+
+            <Canvas id="f-canvas" camera={{}}>
+                <StudioLights />
+                <ambientLight intensity={0.5} />
+                <ModelScroll />
+            </Canvas>
+
+            <div className="absolute inset-0">
+                {features.map((feature, index) => (
+                    <div className={clsx('box', `box${index + 1}`, feature.styles)}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)